xarray/core/ops.py

```diff
@@ -475,7 +475,14 @@ def inject_all_ops_and_reduce_methods(cls, priority=50, array_only=True):
 def inject_bottleneck_rolling_methods(cls):
     # standard numpy reduce methods
     for name in NAN_REDUCE_METHODS:
-        f = getattr(np, name)
+        func_name = 'nan' + name
```

Comment:
Can we reuse these functions instead of writing this all again?
https://github.com/jhamman/xarray/blob/rolling_with_nans/xarray/core/ops.py#L293
https://github.com/jhamman/xarray/blob/rolling_with_nans/xarray/core/ops.py#L389

Yes, I know it's heavy on the magic :)